backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -410,14 +393,22 @@
   ::data)
 
 (defmethod ig/init-key :akvo.lumen.component.keycloak/keycloak  [_ {:keys [credentials data] :as opts}]
+  (log/info "Starting keycloak")
   (let [{:keys [issuer openid-config api-root] :as this} (keycloak (assoc data :credentials credentials))
         http-timeout 10000
-        openid-config (fetch-openid-configuration issuer {:timeout http-timeout})]
+        cm (http.conn-mgr/make-reusable-conn-manager {:timeout 2 :threads 3 :insecure? false :default-per-route 10})
```

Comment:
`timeout` is actually the TTL of the connection. We can be more generous here. I would suggest 20 or 30 secs.